frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -105,12 +108,33 @@ export const filterRevisionsByActiveApplication = (
   return filteredRevisions;
 };
 
+const getBuildConfigforKnativeService = (
+  resource: K8sResourceKind,
+  resources: TopologyDataResources,
+): BuildConfigOverviewItem[] => {
+  const service = resource.metadata.labels?.['app.kubernetes.io/instance'];
+  const buildConfigs = resources.buildConfigs.data.filter(
+    (res) => res.metadata.labels?.['app.kubernetes.io/instance'] === service,
+  );
```

Comment:
Instead of labels, can we use more heuristics similar to other workloads for relating the the build configs?

Reply:
@christianvogt I couldn't find a way to associate buildConfigs to knative service like we have for other resources

cc: @invincibleJai